salt/states/module.py

```diff
@@ -1,12 +1,11 @@
-# -*- coding: utf-8 -*-
 r"""
 Execution of Salt modules from within states
 ============================================
 
 .. note::
 
     There are two styles of calling ``module.run``. **The legacy style will no
-    longer be available starting in the 3005 release.** To opt-in early to the
+    longer be available starting in the 3001 release.** To opt-in early to the
```

Comment:
this should remain 3005

Reply:
Fixed.

Reply:
I'm not seeing this resolved or the changelog entry. Also looks like pre-commit is failing now.

Reply:
You're right - I didn't push the changes properly. It should be fixed now